DEBUG (07/04/2024 21:06:55): server init (Line: 37 [server.py])
DEBUG (07/04/2024 21:07:00): Connection established from ('127.0.0.1', 33134) (Line: 25 [server.py])
DEBUG (07/04/2024 21:07:05): client: 127.0.0.1 - ECHO (Line: 125 [commands.py])
DEBUG (07/04/2024 21:07:08): client: 127.0.0.1 - TIME (Line: 125 [commands.py])
DEBUG (07/04/2024 21:07:11): client: 127.0.0.1 - LS (Line: 125 [commands.py])
DEBUG (07/04/2024 21:07:30): Connection lost with ('127.0.0.1', 33134) (Line: 60 [server.py])
